src/pyhf/infer/mle.py

```diff
@@ -96,9 +96,11 @@ def fit(data, pdf, init_pars=None, par_bounds=None, fixed_params=None, **kwargs)
     Args:
         data (:obj:`tensor`): The data
         pdf (~pyhf.pdf.Model): The statistical model adhering to the schema model.json
-        init_pars (:obj:`list`): Values to initialize the model parameters at for the fit
-        par_bounds (:obj:`list` of :obj:`list`\s or :obj:`tuple`\s): The extrema of values the model parameters are allowed to reach in the fit
-        fixed_params (:obj:`list`): Parameters to be held constant in the fit.
+        init_pars (:obj:`list` of :obj:`float`): The starting values of the model parameters for minimization.
+        par_bounds (:obj:`list` of :obj:`list`/:obj:`tuple`): The extrema of values the model parameters
+            are allowed to reach in the fit.
```

Comment:
Should we talk about expected shape here too?

Reply:
I was asking here because it is a bit harder to talk about shape given that it is a list of objects, but I think this is fine. People will probably complain if things are unclear.